app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/pocket/PocketStoriesComposables.kt

```diff
@@ -116,11 +121,15 @@ fun PocketStories(
                 columnItems.forEachIndexed { rowIndex, story ->
                     if (story == placeholderStory) {
                         ListItemTabLargePlaceholder(stringResource(R.string.pocket_stories_placeholder_text)) {
-                            onExternalLinkClicked("http://getpocket.com/explore")
+                            onExternalLinkClicked("https://getpocket.com/explore?$POCKET_FEATURE_UTM_KEY_VALUE")
                         }
                     } else {
+                        val uri = Uri.parse(story.url)
```

Comment:
One idea here to wrap this as an extension function and add a test for this since it's a bit important that we ensure we are building the right URL. For example, `story.url.(to|as)[PocketStories]UTMLink()`

Reply:
An extension to String seems excessive. It would just pollute it's namespace with something that is particular to this usecase. We can have the extension private to this file but there would be no guarantee that method is called.
I've created https://github.com/mozilla-mobile/fenix/issues/21680 for testing all composables.